common/src/ipc/transmit.rs

```diff
@@ -181,9 +191,10 @@ impl From<RawMsg> for Answer {
 }
 // TODO: end remove ugly mess block
 
-macro_rules! code {
+macro_rules! msg {
     ($($name:ident $num:literal),* $(,)?) => {
-        pub enum Code {
+        #[derive(Copy, Clone, PartialEq, Eq)]
```

Comment:
I don't know if it would be worth it for another dependency, but the `strum` crate has derive macros that can do this conversion between a repr for you (`FromRepr`) - also, if you mark `Code` as `#[repr(u64)]`, you can just do e.g. `Code::ReqPing as u64` for free without needing the `into` fn.

Reply:
I have to convert back from `u64`, which (to my knowledge) cannot be done with `num as Code`. Also, owner of this project expressed his desire not to add dependencies.

Reply:
Like I said before, I specially don't like pulling in dependencies just to use a single function or feature, a single time. In fact, if I was writing this PR, I wouldn't even have used a macro at all. I would have instead written everything manually, though I can see the benefits of the macro @rkuklik wrote.